sources/packages/api/src/services/education-program/education-program.service.ts

```diff
@@ -172,13 +172,17 @@ export class EducationProgramService extends RecordDataModelService<EducationPro
     program.name = educationProgram.name;
     program.description = educationProgram.description;
     const auditUser = { id: auditUserId } as User;
+    const now = new Date();
     if (!programId) {
       // Institution should never be updated after creation.
       program.institution = { id: institutionId } as Institution;
       program.submittedBy = auditUser;
+      program.submittedDate = now;
       program.creator = auditUser;
+      program.createdAt = now;
     } else {
       program.modifier = auditUser;
+      program.updatedAt = now;
```

Comment:
for else do we need this? it will automatically updated, as you did in draft applications

Reply:
Right, I missed it.